src/migrator/utils.ts

```diff
@@ -158,3 +145,14 @@ WHERE "migration"."model name" = ${1}`,
 
 	return JSON.parse(data.executed_migrations) as string[];
 };
+
+export const migrationTablesExist = async (tx: Tx) => {
+	return (
+		await Promise.all(
+			['migration', 'migration lock'].map(async (tableName) => {
+				const result = await tx.tableList(`name = '${tableName}'`);
+				return result.rows.length > 0;
+			}),
+		)
+	).every((check) => check);
```

Comment:
I think it'd be better to do this as a single query, eg:
```suggestion
	const tables = ['migration', 'migration lock'];
	const where = tables.map((tableName) => `name = '${tableName}'`).join(' OR ');
	const result = await tx.tableList(where);
	return result.rows.length === tables.length;
```